### ISSUE TITLE
Standardize units

### ISSUE THREAD

**Issue by nasimrahaman** [pg2455/covid_p2p_simulation#21]
The new mobility engine uses [pint](https://pint.readthedocs.io/en/0.10.1/) to manage units. Since we want to integrate our code with GIS systems (open-street-map), it makes sense to standardise unit handling across the code-base. A good place to start would be in `config.py` -- for instance, I can't tell what [`MIN_DIST_ENCOUNTER = 20`](https://github.com/pg2455/covid_p2p_simulation/blob/master/config.py#L78) means. It's probably not the units used to measure the size of the city (where a default city is a square of length 1000). This could be `MIN_DIST_ENCOUNTER = 20 * CM` where `CM` is supplied by pint.

**Comment by scharmchi** [comment]
I agree. I also have a hard time understanding time measurements. Whether a number is hours or minutes is unclear to me.

**Comment by pg2455** [comment]
It would be such a good idea to use `pint`. It definitely will remove a lot of sources of confusion. We should put it higher in our priority list to prevent future confusion.
